### ISSUE TITLE
Number of mails to sync via "Load up to 25 more"

### ISSUE THREAD

**Issue by smoneck** [k9mail/k-9#2154]
**Account settings** --> **Fetching mail** --> **Local folder size** sets the number of mails to fetch. Scrolling down in mailbox, you find **Load up to 25 more**, what let you fetch n x 25 more mails than stated in **Local folder size**.

### Expected behavior
When I delete a mail, I expect it to be subtracted from the amount of mails added to **Local folder size**.
Say, I have **Local folder size** == 25. When I extend fetching to 200 mails via **Load up to 25 more** and delete 197 mails, I want the fetching to fall back to 25.

### Actual behavior
Instead it continues the 200 mail synchronization until 197 new mails came in and kind of flushed the old ones out.

### Steps to reproduce
See above.

### Why this is important/annoying to me
I get lots of "spam" (not really important feedback of scripts). When I use **Load up to 25 more** several times to extend the fetching up to the last important mail and then delete the junk inbetween, it continues to fetch 200 mails, whats heavy on my data-bill and completely unnecessary since I know the older ones.

### Environment
K-9 Mail version: 5.203

Android version: 7.1.1

Account type (IMAP, POP3, WebDAV/Exchange): IMAP

**Comment by philipwhiuk** [comment]
If you drag down from the top it will refresh to the last 25.

**Comment by smoneck** [comment]
I just fetched 50 mails, deleted 32, draged down from the top and it fetched 50 mails again.

**Comment by philipwhiuk** [comment]
Yeah, hence I why I deleted my comment and re-opened it. It certainly ought to reset back to 25 (and it did when I tried once, oddly), but it seems to not be doing it all the time :(

**Comment by olifre** [comment]
Same here :-( Any known workarounds?

**Comment by jgarcke** [comment]
A workaround is to change the settings Local folder size to a different value and change the settings back. I tried 25 -> 10 -> 25 and 25 -> 50 -> 25, both worked.
